css-ecommerce-page/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Ecommerce Challenge</title>
+  <link href="https://fonts.googleapis.com/css?family=DM+Sans&display=swap" rel="stylesheet">
+  <link href="styles.css" rel="stylesheet">
+</head>
+<body>
+  <div class="container">
+    <div class="row justify-center header-text">
+      <h3 class="header-text">New Arrivals</h3>
+    </div>
+
+    <div class="row justify-center">
+      <div class="cloumn-fifth">
```

Comment:
`column-fifth`

This class name is misspelled on each of your `column` class elements here. The dangers of copy-paste 😉